.buildkite/pipeline.yml

```
@@ -107,4 +107,42 @@ steps:
     depends_on:
       - lint
       - unit
-      - test_network
\ No newline at end of file
+      - test_network
+  - label: test_byoc_aws
+    key: test_byoc_aws
+    if: build.pull_request.labels includes 'byoc-ready'
+    command: make test_byoc_aws
+    plugins:
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - secret-id: sdlc/prod/buildkite/devex_ci_creds_aws
+            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
+      - docker#v5.12.0:
+          image: "golang:1.22"
+          environment:
+            - INTEGRATION_PROVIDER_SECRET_REDPANDA_CLIENT_ID
+            - INTEGRATION_PROVIDER_SECRET_REDPANDA_CLIENT_SECRET
+            - TF_LOG=DEBUG
+            - "AWS_ACCESS_KEY_ID=$$DEVEX_CI_CREDS_AWS_ACCESS_KEY_ID"
+            - "AWS_SECRET_ACCESS_KEY=$$DEVEX_CI_CREDS_AWS_SECRET_ACCESS_KEY"
+    depends_on:
+      - lint
+      - unit
+  - label: test_byoc_gcp
+    key: test_byoc_gcp
+    if: build.pull_request.labels includes 'ci-ready'
+    command: make test_byoc_gcp
+    plugins:
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - secret-id: sdlc/prod/buildkite/devex_ci_creds_aws
+      - seek-oss/aws-sm#v2.3.2:
+          json-to-env:
+            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
```

```suggestion
      - seek-oss/aws-sm#v2.3.2:
          json-to-env:
            - secret-id: sdlc/prod/buildkite/devex_ci_creds_gcp
            - secret-id: sdlc/prod/buildkite/integration_provider_secret_redpanda_client
```

i think you want to use gcp creds here instead of aws creds